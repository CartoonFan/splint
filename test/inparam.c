extern unknown (int *x);
extern inparam (/*@in@*/ int *x);

void f (/*@out@*/ int *x)
{
  if (3 > 4)
    {
      unknown (x);
    }
  else
    {
      inparam (x);
    }
}
