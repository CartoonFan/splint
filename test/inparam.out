
Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.
inparam.c: (in function f)
inparam.c:8:16: Passed storage x not completely defined (*x is undefined):
                   unknown (x)
inparam.c:12:16: Passed storage x not completely defined (*x is undefined):
                    inparam (x)

Finished LCLint checking --- 2 code errors found, as expected

Cannot find standard library: ansi.lcd
     Check LARCH_PATH environment variable.
inparam.c: (in function f)
inparam.c:12:16: Passed storage x not completely defined (*x is undefined):
                    inparam (x)

Finished LCLint checking --- 1 code error found, as expected
